### PR TITLE
Feature/use new marketplace lite component

### PR DESCRIPTION
**Pull request by circlecube**
## Proposed changes

This is an example of how to use the new marketplaceLite component.

## Type of Change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/bluehost/.github/blob/master/.github/contributing.md) doc
- [ ] Linting and tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/app/pages/ecommerce/index.js [hunk 1]
```diff
@@ -29,13 +30,22 @@ function EcommercePage() {
...
     <BWACommonTemplate>
+      
+      <EcomMarketplace />
+
       <NewfoldECommerce
         state={eCommerceState}
         actions={eCommerceActions}
         wpModules={wpModules}
         section={section}
+        marketplace={<EcomMarketplace />}
```
[the remark this answers is not in the document]
Comment (circlecube) [reply src/app/pages/ecommerce/index.js:48]: Yes, that's even better! Good call, thanks for the suggestion.

**Review by aulisius — APPROVED** [review]

File: src/app/pages/ecommerce/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (circlecube) [reply src/app/pages/ecommerce/index.js]: This is in place in the wpModules now rather than as an individual prop.

**Comment by circlecube** [comment]
This PR was created as an example of how to integrate the new marketplace lite component. Using this rather than the full marketplace component, you won't need to include the code to hide the tabs etc. I'll leave it as a draft PR for example, and once you have updated the module to use this component as is shown here, we can close this PR and remove the branch. Thanks @aulisius

**Comment by circlecube** [comment]
Closing this branch since it was example code and has been implemented on the module side.

**Pull request closed without merging**